### PR TITLE
Reformat tutorials and delete docker

### PR DESCRIPTION
**Pull request by benoit9126**
- [x] Reformat tutorials notebook into markdown files
- [x] Split the "Advanced tutorial" into several files
- [x] Rename short circuit into short-circuit everywhere
- [x] Delete the notebooks
- [x] Delete the docker container build (too complicated and useless with a working conda installation)

### CODE REVIEW

File: doc/usage/Connecting_Elements.md [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+```
+
+```{danger}
+The load element `load` doesn't belong to a network and a part of its results is not accessible any more. `res_`
+methods may raise errors.
+```
+
+## Connecting  an element
+
+The connection of an element to an existing network is automatically done when the elements are built. Let's create
```
Comment (alihamdan) [inline doc/usage/Connecting_Elements.md:101]: As the example below demonstrates
```suggestion
The connection of an element to an existing network is not automatically done when the elements are built. Let's create
```

File: doc/usage/Flexible_Loads.md [hunk 2]
```diff
@@ -0,0 +1,242 @@
...
+>>> abs(load_bus3.res_voltages)
+array([243.08225748, 232.46046866, 233.62854073]) <Unit('volt')>
+```
+
+The actually produced power of the flexible load is a result of the computation and can be
+accessed using the `res_flexible_powers` property of the load. Here, one can note that:
+
+* The active power for the phase `'a'` is negative meaning production;
+* The actual value of this active power is lower that the one requested as the control was activated;
+* The power for phases `'b'` and `'c'` is 0 VA as expected.
```
Comment (alihamdan) [inline doc/usage/Flexible_Loads.md:182]: This section (starting from "Here, one can ...") should be moved below the code block that shows the example below

File: doc/usage/Flexible_Loads.md [hunk 3]
```diff
@@ -0,0 +1,242 @@
...
+>>> en.solve_load_flow(auth=auth)
+6
+>>> abs(load_bus3.res_voltages)
+array([239.5133208 , 230.2108052 , 237.59184615]) <Unit('volt')>
+```
+
+One can note that this time, the phase `'a'` consumes reactive power to limit the voltage rise in
+the network. Moreover, the magnitude of the power on phase `'a'` is approximately $4 kVA$ which is
+the maximum allowed apparent power for `load3`. In order to maintain this maximum, a
+[Euclidean projection](models-flexible_load-projection) has been used.
```
Comment (alihamdan) [inline doc/usage/Flexible_Loads.md:237]: Same here, move below the code block

File: doc/usage/Short_Circuit.md [hunk 4]
```diff
@@ -0,0 +1,170 @@
...
+| line1     | b     | -376.14-74.96j |      376.12+74.96j | 58424.25+66571.96j |     -41140.25-59810.05j |    -115.47-200j |    -74.72-173.91j |
+| line1     | c     |    -0.49-0.42j |         0.49+0.21j |        -26.73-147j |           -14.9+126.71j |    -115.47+200j |   -117.06+208.26j |
+| line1     | n     |      -0.1+0.1j |            -0.1-0j |                 0j |             -0.15+0.85j |              0j |         1.63-8.2j |
+| line2     | a     |  376.51+75.17j | **-376.45-74.93j** | 69627.17-24139.23j | **-14217.89+41992.79j** |   190.15-26.15j | **57.69-100.07j** |
+| line2     | b     | -376.12-74.96j |  **376.45+74.93j** | 41140.25+59810.05j |  **14217.89-41992.79j** |  -74.72-173.91j | **57.69-100.07j** |
+| line2     | c     |    -0.49-0.21j |                 0j |       14.9-126.71j |                      0j | -117.06+208.26j |   -120.25+224.73j |
+| line2     | n     |         0.1+0j |                 0j |         0.15-0.85j |                      0j |       1.63-8.2j |        4.88-24.6j |
+
+Looking at the line results of the second bus of the line "line2", which is "bus2" where we added the short-circuit,
```
Comment (alihamdan) [inline doc/usage/Short_Circuit.md:77]:
```suggestion
Looking at the line results of the second bus of the line `line2`, which is `bus2` where we added the short-circuit,
```

File: doc/usage/Short_Circuit.md [hunk 5]
```diff
@@ -0,0 +1,170 @@
...
+
+```pycon
+>>> en.res_grounds
+```
+
+| ground_id | potential |
+|:----------|----------:|
+| gnd       |        0j |
+
+Here the potential at phase "a" of bus "bus2" is zero, equal to the ground potential. The sum of the currents in the
```
Comment (alihamdan) [inline doc/usage/Short_Circuit.md:146]:
```suggestion
Here the potential at phase "a" of bus `bus2` is zero, equal to the ground potential. The sum of the currents in the
```

**Review by alihamdan — COMMENTED** [review]
Looks good. I left some suggestions.

**Review by alihamdan — APPROVED** [review]

File: doc/usage/Getting_Started.md [hunk 6]
```diff
@@ -0,0 +1,475 @@
...
+6. [Load the saved network and the results from the disk](gs-loading-network).
+
+(gs-creating-network)=
+
+## Creating a network
+
+An electrical network can be built by assembling basic elements provided in [roseau.load_flow.models][api-models].
+The following is a summary of the available elements:
+
+* Buses:
```
Comment (Saelyos) [inline doc/usage/Getting_Started.md:23]: The link to `[api-models]` already exists after the short description of the elements. At this points, I think I would prefer to have a link to `/models`

File: doc/usage/Getting_Started.md [hunk 7]
```diff
@@ -0,0 +1,475 @@
...
+  *Goldstein and Price* linear search;
+* `solver_params`: the parameters used by the solver;
+* `tolerance`: the requested tolerance for the solver. $10^{-6}$ is the default;
+* `max_iterations`: the requested maximum number of iterations for the solver. 20 is the default;
+* `warm_start`: if `True`, the results (potentials of each bus) from the last valid run are used
+  as a starting point for the solver. For large networks, using a warm start can lead to performance gains as the
+  solver will converge faster. `True` is the default;
+* `status`: the convergence of the load flow. Two possibilities: *success* or *failure*;
+* `iterations`: the number of iterations made by the solver.
+* `residual`: the precision which was reached by the solver (lower than the tolerance if successful solve).
```
Comment (Saelyos) [inline doc/usage/Getting_Started.md:172]: Maybe add at the bottom a link to `/solvers` ?

File: doc/usage/Getting_Started.md [hunk 8]
```diff
@@ -0,0 +1,475 @@
...
+... ground = Ground(id="gnd")
+... # Fix the potential of the ground at 0 V
+... pref = PotentialRef(id="pref", element=ground)
+... ground.connect(source_bus, phase="n")
+
+>>> # Create a LV source at the first bus
+... # Volts (phase-to-neutral because the source is connected to the neutral)
+... un = 400 / np.sqrt(3)
+... source_voltages = [un, un * np.exp(-2j * np.pi / 3), un * np.exp(2j * np.pi / 3)]
+... vs = VoltageSource(id="vs", bus=source_bus, phases="abcn", voltages=source_voltages)
```
Comment (Saelyos) [inline doc/usage/Getting_Started.md:82]: For all the voltage sources, lines and loads, the phases have been specified which is not mandatory. IMO the phases should not be specified in those cases, as there is more chance to do something wrong (for exemple adding an MV line with `"abc"` in a LV network `"abcn"`) without noticing. If the phases are not specified the user will get an error. The phases for those elements should only be done for special cases.

**Review by Saelyos — CHANGES_REQUESTED** [review]

**Review by Saelyos — APPROVED** [review]

**Pull request merged**